Styles/component/_header.css

```diff
@@ -0,0 +1,6 @@
+header {
```

Comment:
If you add this to it's own component, it should be a class.
```suggestion
.header {
```